.env.example

```diff
@@ -2,13 +2,14 @@ AUTHENTICATION_FALLBACK=
 BIG_QUERY_API_JSON_KEY=
 BIG_QUERY_DATASET=
 DATABASE_URL=
+DFE_SIGN_IN_HIRING_STAFF_ROLE_ID=
```

Comment:
Should this be `PUBLISHER` rather than hiring staff to match what we call them in most places in the app?

Reply:
OK